starter-code/styles/modules.css

```diff
@@ -2,13 +2,17 @@
   background: #FFEAAA;
 }
 
-/* REVIEW: Float the main header to the left, to make room for the hamburger menu. */
 .heading-group-h1 {
   float: left;
   padding: 0 0 .2em 1em;
 }
 
-/* TODO: Add styles for the icon-menu class. It should be bigger, and off to the right. */
+.icon-menu {
+  float: right;
+  font-size: 500%;
```

Comment:
Your hamburger menu is huge! That is clever because phone screens are small, so a bigger hamburger menu makes it easier to click!